[2025-02-22T14:56:29.319-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-22T14:56:29.342-0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fii_info.store.store scheduled__2025-02-22T17:40:00+00:00 [queued]>
[2025-02-22T14:56:29.357-0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fii_info.store.store scheduled__2025-02-22T17:40:00+00:00 [queued]>
[2025-02-22T14:56:29.357-0300] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-22T14:56:29.383-0300] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): store.store> on 2025-02-22 17:40:00+00:00
[2025-02-22T14:56:29.388-0300] {standard_task_runner.py:72} INFO - Started process 40829 to run task
[2025-02-22T14:56:29.394-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'fii_info', 'store.store', 'scheduled__2025-02-22T17:40:00+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/info_Fii.py', '--cfg-path', '/tmp/tmpwm0g16pe']
[2025-02-22T14:56:29.398-0300] {standard_task_runner.py:105} INFO - Job 58: Subtask store.store
[2025-02-22T14:56:29.474-0300] {task_command.py:467} INFO - Running <TaskInstance: fii_info.store.store scheduled__2025-02-22T17:40:00+00:00 [running]> on host LAPTOP-3G01DURD.
[2025-02-22T14:56:29.651-0300] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fii_info' AIRFLOW_CTX_TASK_ID='store.store' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T17:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-22T17:40:00+00:00'
[2025-02-22T14:56:29.653-0300] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-22T14:56:29.672-0300] {python.py:240} INFO - Done. Returned value was: None
[2025-02-22T14:56:29.678-0300] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-22T14:56:29.678-0300] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fii_info, task_id=store.store, run_id=scheduled__2025-02-22T17:40:00+00:00, execution_date=20250222T174000, start_date=20250222T175629, end_date=20250222T175629
[2025-02-22T14:56:29.768-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-22T14:56:29.790-0300] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-22T14:56:29.799-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
